planning/planning_scene_monitor/include/moveit/planning_scene_monitor/current_state_monitor.h

```diff
@@ -59,8 +59,10 @@ class CurrentStateMonitor
   /** @brief Constructor
    *  @param robot_model The current kinematic model to build on
    *  @param tf A pointer to the tf transformer to use
+   *  @param nh An optionnal node handle to pass options
    */
```

Comment:
I was positivily sure that this was the right way to spell this word. I'll change it.